images/pulp/Containerfile

```
@@ -0,0 +1,16 @@
+ARG PULPCORE_VERSION=nightly
+
+FROM quay.io/centos/centos:stream9
+
+ARG PULPCORE_VERSION
+
+RUN dnf -y update && \
```

Not blocking, but this is something we should agree on in all our containers. I'd be tempted to say that we shouldn't update everything and instead rely on container images to be updated
```suggestion
RUN \
```

---

I wouldn't want to leave our users potentially exposed if the base image cadence is long.